client/my-sites/post-share/index.jsx

```diff
@@ -339,9 +351,24 @@ class PostShare extends Component {
 
 		return (
 			<div className="post-share__services">
-				<h5 className="post-share__services-header">
-					{ translate( 'Connected services' ) }
-				</h5>
+				<SectionHeader className="post-share__services-header" label={ translate( 'Connected accounts' ) }>
+					<Button
+						compact
+						href={ '/sharing/' + siteId }
+						className="post-share__add-button"
+						onMouseEnter={ this.showAddTooltip }
+						onMouseLeave={ this.hideAddTooltip }
+						ref="addAccountButton"
+						aria-label={ translate( 'Add account', { context: 'button label' } ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Add account' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.